dozer-types/src/types/field.rs

```diff
@@ -769,6 +772,240 @@ pub fn field_test_cases() -> impl Iterator<Item = Field> {
     .into_iter()
 }
 
+pub fn arrow_field_test_cases() -> impl Iterator<Item = Field> {
```

Comment:
Let's implement this function with using `Iterator::filter` to remove the fields that are not supported by arrow, instead of copy the fields.